docs/source/getting-started.md

```diff
@@ -0,0 +1,21 @@
+# Getting started
+
+`menuinst` can be used to create shortcuts or menu items across operating systems.
+It's designed to integrate nicely with `conda` packages, 
+but it can be used without it to an extent.
```

Comment:
To wht extent 🤔 ?

Reply:
From the Python API, it needs to be studied :)